app/Http/Controllers/Site/HomePageController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Http\Controllers\Site;
+
+class HomePageController extends \App\Http\Controllers\BaseController
+{
+    public function view()
```

Comment:
Could you use the __invoke() function on this controller, given it is a single action?

Reply:
Good spot!